card/gem_mods/ace_editor/card-mod-ace_editor.gemspec

```diff
@@ -0,0 +1,29 @@
+# -*- encoding : utf-8 -*-
+
+version = File.open(File.expand_path("../../../../card/VERSION", __FILE__)).read.chomp
+vbits = version.split('.').map &:to_i
+vplus = { 0 => 90, 1 => 100 } # can remove and hardcode after 1.0
+vminor = vplus[ vbits[0] ] + vbits[1]
+card_version = [1, vminor, vbits[2]].compact.map(&:to_s).join "."
+# see card.gemspec for explanation of all of this, which has been ham-handedly
+# cut and pasted here.
+
+Gem::Specification.new do |s|
+  s.name = "card-mod-ace_editor"
+  s.version = card_version
```

Comment:
we actually just want "version", not card_version.  card_version is going to be weird until we get to decko 1.0.  Then every gem will just use the same version.

Reply:
Ok, I'll change that. I wasn't sure how that would work with the other version streams.